pyproject.toml

```diff
@@ -44,7 +49,7 @@ skip_gitignore = true
 [tool.black]
 line-length = 120
 skip-string-normalization = true
-target-version = ['py37', 'py38', 'py39']
+target-version = ['py312']
```

Comment:
dont want to force 312

Reply:
reverted in d8c8fd8ecfb0d078a87d47d9858e550f5ed35aa0